### PR TITLE
Update airbase to 106

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

File: testing/trino-benchto-benchmarks/pom.xml [hunk 1]
```diff
@@ -85,7 +85,9 @@
             <plugin>
                 <artifactId>maven-assembly-plugin</artifactId>
                 <configuration>
-                    <descriptor>src/assembly/benchmarks.xml</descriptor>
+                    <descriptors>
```
Comment (sopel39) [inline testing/trino-benchto-benchmarks/pom.xml:88]: undo?

**Review by sopel39 — APPROVED** [review]

File: testing/trino-benchto-benchmarks/pom.xml (hunk 1 above)
Comment (wendigo) [reply testing/trino-benchto-benchmarks/pom.xml:88]: It's intentional. Plugin was updated :)

File: client/trino-jdbc/src/test/java/io/trino/jdbc/BaseTestJdbcResultSet.java [hunk 2]
```diff
@@ -752,12 +752,12 @@ public void testArray()
             // no NULL elements
             checkRepresentation(connectedStatement.getStatement(), "ARRAY[1, 2]", Types.ARRAY, (rs, column) -> {
                 Array array = rs.getArray(column);
-                assertThat(array.getArray()).isEqualTo(new int[] {1, 2});
+                assertThat(array.getArray()).isEqualTo(new Object[] {1, 2});
```
Comment (wendigo) [inline client/trino-jdbc/src/test/java/io/trino/jdbc/BaseTestJdbcResultSet.java:755]: getArray returned array of objects (being integers). Previously this assertion worked but I guess that it was fixed in the AssertJ core - that's why I've made this change.

File: core/trino-parser/pom.xml [hunk 3]
```diff
@@ -48,7 +48,6 @@
         <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
-            <version>3.13.2</version>
```
Comment (wendigo) [inline core/trino-parser/pom.xml:51]: I've removed the fixed version as it caused problems

**Comment by sopel39** [comment]
there are failures

**Review by losipiuk — APPROVED** [review]

**Pull request merged**